docs/source/onedal/algorithms/decomposition/pca.rst

```diff
@@ -33,68 +33,89 @@ Mathematical formulation
 method such as [Ping14]_.
 
-The final step is to sort the set of pairs :math:`(\lambda_i, \upsilon_i)` in
-the descending order by :math:`\lambda_i` and form the resulting matrix :math:`T
-= (\upsilon_{i,1}, \cdots, \upsilon_{i,r}), \quad 1 \leq i \leq p`.
-Additionally, the means and variances of the initial dataset are returned.
+In the final step, the eigenvalues (:math:`\lambda_k`) are sorted in descending order 
+to determine the order of the principal components. Each principal component is 
+stored as a row of the final resulting matrix, :math:`T = (\upsilon_{1j}, \cdots, \upsilon_{rj}), \quad 1 \leq j \leq p`.
+Additionally, the means and variances of the input dataset are returned.
 
 .. _pca_t_math_svd:
 
 Training method: *SVD*
 ~~~~~~~~~~~~~~~~~~~~~~
 
-This method uses singular value decomposition of the dataset to compute its
-principal components. The method relies on the following steps:
+The singular value decomposition (SVD) is a matrix factorization technique that
+decomposes an observation matrix :math:`X_{n \times p}` into three matrices as :math:`X = U\SigmaV^*`.
+Here, the columns of :math:`U` are the left-singular vectors, the
+columns of  :math:`V` are the right-singular vectors, :math:`V^*` is the 
+conjugate transpose of the matrix :math:`V` and the diagonal entries of :math:`\Sigma` 
+are the singular values (:math:`\sigma`) of :math:`X`. The right-singular vectors are 
+the principal components of :math:`X`. The steps of computing principal components using SVD technique
+are as follows:
+
+#. Mean centering the input data
+#. Decomposing the mean-centered input data to compute the singular values and the singular vectors
+#. Processing(sorting and storing) the results
 
-#. Computation of the singular values and singular vectors
-#. Formation of the matrices storing the results
+First step is to mean center the input data :math:`M = M_{ij}`, where :math:`M_{ij} = X_{ij} - \frac{\sum_{i=1}^n X_{ij}}{n}`.
 
-To compute singular values :math:`\lambda_i` and singular vectors :math:`u_i`
-and :math:`v_i`, the implementer can choose an arbitrary method such as
-[Demmel90]_.
+Singular values :math:`\sigma_k`, left-singular vectors :math:`U_k` and right-singular vectors :math:`V_k` of matrix :math:`M` can be computed with an arbitrary method like the one described in [Demmel90]_.
 
-The final step is to sort the set of pairs :math:`(\lambda_i, v_i)` in the
-descending order by :math:`\lambda_i` and form the resulting matrix :math:`T =
-(v_{i,1}, \cdots, v_{i,r}), \quad 1 \leq i \leq p`. Additionally, the means and
+The final step is to find a permutation matrix :math:`Q_{p \times p}` such that the diagonal entries of :math:`\SigmaQ`are sorted in a descending order i.e :math:`\sigma_k \geq \sigma_{k+1}, for all k < p assuming n > p`.
+The rows of the resulting matrix :math:`T = V^*Q` are the principal components of :math:`X`. Note that the rows
```

Comment:
.. note: The rows of :math:`T` are also the eigenvectors of the covariance matrix of :math:`X`.